packages/ui/package.json

```diff
@@ -6,13 +6,13 @@
     "build-storybook:web": "storybook build -c ./src/.storybook-web",
     "lint": "eslint .",
     "lint:fix": "pnpm lint --fix",
-    "prepare": "panda codegen -c ./panda.config.ts",
+    "panda": "panda codegen -c ./panda.config.ts",
```

Comment:
Good idea renaming this 👍

Reply:
Not just rename, `prepare` default runs on `pnpm i` which creates a problem on install as it requires there to be a build first

Reply:
WOW, OK, good catch then!